cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer_test.go

```diff
@@ -14,23 +14,26 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package install
+package fuzzer
 
 import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/testing/roundtrip"
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+	kubeadmfuzzer "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/fuzzer"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 )
 
-func init() {
-	Install(legacyscheme.Scheme)
-}
-
-// Install registers the API group and adds types to a scheme
-func Install(scheme *runtime.Scheme) {
+// installAPIsToScheme registers the API group and adds types to a scheme
+func installAPIsToScheme(scheme *runtime.Scheme) {
```

Comment:
why a local installer func? Why not the one shared by the kubeadm scheme?

Reply:
Good feedback! I'm now using `scheme.AddToScheme` instead, with the extra registrations added